Update test cases for #752 and add test cases for #894

jordanpadams opened the pull request:
## 🗒️ Summary
Update test cases for #752 and add test cases for #894

## ⚙️ Test Data and/or Report
See test execution

## ♻️ Related Issues
<!--
    Reference related issues here and use `Fixes` or `Resolves` in order to automatically close the issue upon merge. For more information on autolinking to tickets see https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls.

    * for issues in this repo:
        - fixes #1
        - fixes #2
        - refs #3
    * for issues in other repos: NASA-PDS/my_repo#1, NASA-PDS/her_repo#2
-->
#752 
#894 
#898

nutjob4life reviewed (commented):
`mvn clean test` fails on `rsjoyner_updates_rebasedx2`:
```
$ git checkout rsjoyner_updates_rebasedx2
$ mvn clean test
…
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   NASA-PDS/pds4-information-model#829 failed. Expected: 5 errors/warnings. Actual: 7 ==> expected: <5> but was: <7>
[ERROR]   Required that internal errors do not occur. ==> expected: <0> but was: <2>
[INFO] 
[ERROR] Tests run: 26, Failures: 2, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for PDS4 Information Model 15.3.0-SNAPSHOT:
[INFO] 
[INFO] PDS4 Information Model ............................. SUCCESS [  0.398 s]
[INFO] Model DM-Document Maven Plugin ..................... SUCCESS [ 17.246 s]
[INFO] Model Ontology ..................................... SUCCESS [  0.516 s]
[INFO] Local Data Dictionary Tool ......................... FAILURE [03:53 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
…
```

rsjoyner commented:
github829.json -- shows 5 totalErrors which is the expected:

  "summary": {
    "totalProducts": 5,
    "totalErrors": 5,
    "totalWarnings": 4,
    "productValidation": {
      "passed": "4",
      "failed": "2",
      "skipped": "0",
      "total": "5"
    },

jordanpadams commented:
@rsjoyner note the new branch for this PR that we discussed on Slack. I moved your changes from the other branch here, moved to their new location, and updated the IM version to 1O00, but a couple tests are still not passing. I don't know what is going on and can't debug until later in the week.

jordanpadams commented:
I would take a look and see if there is anything obvious.

jordanpadams commented:
also it looks like you have test cases for #898 but that ticket appears to still be opened? Has that already been resolved somewhere or not sure what is happening there?
